editor/js/ui/view.js

```diff
@@ -2038,7 +2038,14 @@ RED.view = (function() {
                             .attr("ry",4)
                             .attr("width",16)
                             .attr("height",node_height-12)
-                            .attr("fill",function(d) { return RED.utils.getNodeColor(d.type,d._def); /*d._def.color;*/})
+                            .attr("fill", function (d) {
+                                var nodeColor = RED.utils.getNodeColor(d.type,d._def); /*d._def.color;*/
```

Comment:
I think it would be better to have `RED.utils.getNodeColor` return a default colour, rather than have to add this same logic in multiple places.

Reply:
@knolleary That's right. I'll add the logic into getNodeColor() in editor/js/ui/utils.js.